internal/testdata/TestBuild_with_template

```diff
@@ -1,211 +1,210 @@
-                }
-            ],
-            "telephones": [
-                {
-                    "label": "mobile",
-                    "num": "+34644123123"
-                }
-            ]
-        },
         "customer": {
-            "tax_id": {
-                "country": "ES",
-                "code": "B-85905495"
-            },
-            "name": "Autofiscal S.L.",
             "addresses": [
                 {
-                    "num": "1629",
-                    "street": "Calle Diseminado",
+                    "code": "28792",
+                    "country": "ES",
                     "locality": "Miraflores de la Sierra",
+                    "num": "1629",
                     "region": "Madrid",
-                    "code": "28792",
-                    "country": "ES"
+                    "street": "Calle Diseminado"
                 }
             ],
             "emails": [
                 {
                     "addr": "sam.lown@invopop.com"
                 }
-            ]
+            ],
+            "name": "Autofiscal S.L.",
+            "tax_id": {
+                "code": "B-85905495",
+                "country": "ES"
+            }
         },
+        "issue_date": "2021-06-16",
         "lines": [
             {
                 "i": 1,
-                "quantity": "20",
                 "item": {
                     "name": "Development services day rate",
                     "price": "200.00"
                 },
+                "quantity": "20",
                 "sum": "4000.00",
                 "taxes": [
                     {
                         "cat": "VAT",
-                        "code": "STD"
+                        "rate": "standard"
                     },
                     {
                         "cat": "IRPF",
-                        "code": "STD"
+                        "rate": "pro"
                     }
                 ],
                 "total": "4000.00"
             },
             {
                 "i": 2,
-                "quantity": "10",
                 "item": {
                     "name": "Something random",
                     "price": "100.00"
                 },
+                "quantity": "10",
                 "sum": "1000.00",
                 "taxes": [
                     {
                         "cat": "VAT",
-                        "code": "STD"
+                        "rate": "standard"
                     }
                 ],
                 "total": "1000.00"
             },
             {
                 "i": 3,
-                "quantity": "5",
                 "item": {
                     "name": "Additional random services",
                     "price": "34.50"
                 },
+                "quantity": "5",
                 "sum": "172.50",
                 "taxes": [
                     {
                         "cat": "VAT",
-                        "code": "RED"
+                        "rate": "zero"
```

Comment:
To be completely accurate, this should be `reduced` instead of `zero`. Not really much of a problem though.

Reply:
I'm pretty sure that didn't work.